lambdas/service/app.py

```diff
@@ -138,23 +138,16 @@
             aggregates them.
 
             Azul denormalizes and aggregates metadata into several different
-            indices for selected entity types:
-
-             - [projects](#operations-Index-get_index_projects)
-
-             - [samples](#operations-Index-get_index_samples)
-
-             - [files](#operations-Index-get_index_files)
-
-             - [bundles](#operations-Index-get_index_bundles)
+            indices for selected entity types. Metadata entities can be queried
+            using the [`index`](#operations-tag-Index) endpoints.
 
             This set of indexes forms a catalog. There is a default catalog
```

Comment:
```suggestion
            The set of metadata entity indexes form a catalog. There is a default catalog
```
Nit  / suggestion. The set of indexes are no longer being listed above, so `This set ...` seems out of place.